Metro-Records/la-metro-councilmatic
# Migrate docs from wiki to quarto

## Overview

See title. Much of this is a straightforward migration of the existing docs, but modifications have been made where necessary. For example, I've omitted any direct references to the scrapers that would instead be covered in that documentation, and have adjusted sections that discuss technologies/methods that we no longer use here.

- Connects #1158 

### Demo

<img width="1375" alt="image" src="https://github.com/user-attachments/assets/db0fa8c6-b4c4-40b5-acfe-15e1a1cfdb06">

## Testing Instructions
 * Run the docs locally following the instructions in the README
   * Confirm that things look as expected, items described within make sense, and links work
